CONTRIBUTING.md

```diff
@@ -39,10 +39,16 @@ The `plugin` directory contains the source code for all generators in this packa
    - Run generators locally to verify functionality, for example: `npx nx g expo-app`
    - Run unit tests using `npx nx test nx-generators`
 
-3. **Update generator metadata**
+3. **Test package preview (Optional)**
+   - Change version in `plugin/package.json` to an alpha version, for example: `"0.18.1-alpha.1"`
+   - Build package locally: `npm run build`
+   - Release alpha version of the package: `npm run release -- --tag=alpha`
+   - Install the package in your test environment: `npm i @ronas-it/nx-generators@alpha --save-dev`
+
+4. **Update generator metadata**
```

Comment:
I think this point should be before the previous one. Generator metadata is necessary for alpha versions